exporter/clickhouselogsexporter/migrations/000005_attribute_rename.down.sql

```diff
@@ -0,0 +1,32 @@
+RENAME TABLE if exists signoz_logs.logs_attribute_keys TO signoz_logs.logs_atrribute_keys;
+
+
+-- Materialized view
+DROP table if exists attribute_keys_string_final_mv;
+DROP table if exists attribute_keys_int64_final_mv;
+DROP table if exists attribute_keys_float64_final_mv;
```

Comment:
Shouldn't these be `DROP VIEW IF EXISTS` instead of `DROP TABLE IF EXISTS`?

Reply:
Interestingly both works, but let me update it.